site/docs/v1/tech/tutorials/prometheus.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: doc
+title: Prometheus Setup
+description: Learn how to setup Prometheus to work with FusionAuth
+---
+
+== Prometheus Setup
+
+[NOTE.since]
+====
+Available since `1.26.0`
+====
+
+Prometheus is a tool to monitor metrics and setup alerts in the case that some metric has responded negatively for an extended period. Setting this up with FusionAuth is easy.
+
```

Comment:
Do we have a list of available metrics? Or can we point to another API doc that does list those metrics?